src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -305,6 +306,36 @@ export default class DataTable extends Vue {
     this.$emit(DATA_TABLE_EVENTS.SELECTED_ROWS_CHANGE, selectedRowsData);
   }
 
+  _checkIndeterminate(rowData: DataTableRow) {
+    let parent = rowData.parent;
+
+    while (parent) {
+      const parentCheck = document.querySelector(
```

Comment:
@josecarlosgonzalezv  This suggestion has not been addressed

Reply:
Because the way to get a row is from this `rowId` and if you want to set a ref per row it must be dynamically, for example: `checkboxRef1, checkboxRef2...`, and we cannot access in this.$refs to a variable attribute, this must be a static value. For example: `this.$refs.datatable`.

I cannot concat in `this.$refs.MyVariable` that´s the point I think.

If you have a solution or an idea, let me know it :)